teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative('./person')
+
+class Teacher < Person
+  def initialize(specialization)
+    super()
+    @specialization = specialization
```

Comment:
- [ ] Don't forget to remove the super parenthesis, with it, it throws an error "wrong number of arguments". Calling super() calls the superclass (or ancestors) method without any parameter (assuming this method accepts no parameters...)